src/ol-ext/interaction/measure.js

```diff
@@ -250,6 +256,16 @@ ngeo.interaction.Measure.prototype.onDrawStart_ = function(evt) {
   this.sketchFeature = evt.feature;
   this.overlay_.getFeatures().clear();
   this.createMeasureTooltip_();
+
+  this.changeEventKey = this.sketchFeature.getGeometry().on('change',
```

Comment:
The problem may be related to `undefined`. What if you add `goog.asserts.assert(goog.isDef(geometry))` before registering the listener?